### PR TITLE
docs(store): update guide for jasmine tests

### PR DESCRIPTION
**Pull request by mangei**
see comment: https://github.com/ngrx/platform/issues/3264#issuecomment-1047174743

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Error occurs: TS2339: Property 'afterEach' does not exist on type 'Env'.

## What is the new behavior?

Turns out that just calling the global jasmine afterEach method is enough. When run outside a describe block, it is applied to every test case. The null default is required, otherwise if you have tests that don't provide MockStore, you'll get "NullInjectorError: No provider for MockStore!" error.

Source: https://github.com/ngrx/platform/issues/3264#issuecomment-1047174743

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngrx.io/content/guide/migration/v13.md [hunk 1]
```diff
@@ -109,8 +109,8 @@ When using Jasmine, reset the mock store after each test by adding the following
 ```ts
 import { MockStore } from '@ngrx/store/testing';
 
-jasmine.getEnv().afterEach(() => {
-  TestBed.inject(MockStore)?.resetSelectors();
+afterEach(() => {
+  getTestBed().inject(MockStore, null)?.resetSelectors();
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/migration/v13.md:113]: Is the change from `TestBed` to `getTestBed()` required?
If so, could you add an import statement please.

Comment (timdeschryver) [reply projects/ngrx.io/content/guide/migration/v13.md:113]: @mangei pinging you so we can merge this one

Comment (mangei) [reply projects/ngrx.io/content/guide/migration/v13.md:114]: It is probably not required, see https://angular.io/guide/testing-utility-apis.  
But current code example use getTestBed().   
Also your platform uses it that way: https://github.com/ngrx/platform/blob/master/projects/ngrx.io/src/test.ts

I added the import.

**Review by timdeschryver — APPROVED** [review]

**Review by markostanimirovic — APPROVED** [review]

**Comment by markostanimirovic** [comment]
Thank you @mangei!

**Pull request merged**